templates/components/products/product-view.html

```diff
@@ -249,6 +249,14 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
                 {{> components/common/wishlist-dropdown}}
             {{/if}}
         </div>
+
+        <div id="alertBox-success-message" style="display:none">
+            {{> components/common/alert/alert-success (lang 'products.add_to_wishlist_success_message' wishlistPath=urls.account.wishlists.all)}}
```

Comment:
Why do we need to add all wishlists url to specific alert-success message?

Reply:
@BC-tymurbiedukhin this is done in case JS fails to set the correct link to the wishlist to which the product was added. In this case, the user will be shown a list of all wishlists.

Reply:
Could you please describe the scenario when JS could fail?

Reply:
@BC-tymurbiedukhin done!